testbed/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/testbed
 go 1.17
 
 require (
-	github.com/fluent/fluent-logger-golang v1.7.0
+	github.com/fluent/fluent-logger-golang v1.8.0
```

Comment:
Is this expected to be for testbed module only? Should we update the commit/PR title if that's the case?

Reply:
thanks for pointing this out, the script I used to generate this PR missed adding most of the go.sum and go.mod files 😄